charts/rollups-node/script.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Script to convert values from rollups node TOML file to helm chart values YAML file with descriptions
+
+# Define file names
+TOML_FILE="Config.toml"
+TEMPLATE_VALUES_FILE="values.yaml.tpl"
+VALUES_FILE="values.yaml"
+
+# Download the Config.toml file from the rollups-node repository
+wget https://raw.githubusercontent.com/cartesi/rollups-node/main/internal/config/generate/Config.toml
```

Comment:
We should use the tagged URL, so we can target a version of the configuration. I know we don't have a tagged release for this file yet, but we should be prepared.